test/fixtures/stitch_mongo_fixture.js

```diff
@@ -52,11 +52,22 @@ export default class StitchMongoFixture {
     // create an app `test_app`, and authorize a user
     this.admin = new stitch.Admin('http://localhost:7080');
     await this.admin.client.authenticate('apiKey', userData.apiKey.key);
-    let result = await this.admin.apps(userData.group.groupId).create({ name: 'test_app' });
-    this.clientAppId = result.clientAppId;
     const appConfig = EJSON.parse(fs.readFileSync(`${CONF_PATH}/test_app_config.json`, 'utf8'));
-    await this.admin.apps(userData.group.groupId).app(result._id).replace(appConfig);
+    let result = await this.admin.apps(userData.group.groupId).create(appConfig);
+
+    this.clientAppId = result.clientAppId;
+    await this.admin.apps(userData.group.groupId).app(result._id).authProviders().provider('api', 'key').update({});
     this.appKey = await this.admin.apps(userData.group.groupId).app(result._id).apiKeys().create({ name: 'app-key' });
+    await this.admin.apps(userData.group.groupId).app(result._id).services().create({name: 'mdb1', type: 'mongodb'});
```

Comment:
let's DRY up the code from this line below, by using something like 
`const mdb1 = appConfig.services.mdb1`
then you can just refer to `mdb1.config` and `mdb1.rules` thereafter, which should make this look more compact.
Also, assign the two hex strings to constants, like `rule1 = '5873a33f772e2e08ce645b9a', rule2 = ... <etc>` so that we can use them as indexes in the object without repeating the literal values.